native_client/README.rst

```diff
@@ -168,7 +169,7 @@ The path of the system tree can be overridden from the default values defined in
 Android devices
 ^^^^^^^^^^^^^^^
 
-We have preliminary support for Android relying on TensorFlow Lite, with Java and JNI bindinds. For more details on how to experiment with those, please refer to ``native_client/java/README.rst``.
+We have support for Android relying on TensorFlow Lite, with Java and JNI bindinds. For more details on how to experiment with those, please refer to ``native_client/java/README.rst``.
```

Comment:
1. The referenced file, `native_client/java/README.rst`, says "This is still preliminary work" as the very first sentence, so if we're removing that term here and publishing the docs, we should make sure we have fixed whatever was causing this to be considered preliminary, and review the whole documentation.

2. We should move the Java README into ReadTheDocs as well if we want to reference it.

3. The Java README mixes build and usage instructions in a way that is not very clear that you don't have to build things yourself for usage and experimentation. We should re-arrange it to start with simple usage instructions using pre-built binaries, and then have build instructions.

Reply:
Good catches, ill address them.

Reply:
This still has a file path instead of `` :ref:`java-android-bindings` ``